eo-phi-normalizer/src/Language/EO/Phi/Rules/Yaml.hs

```diff
@@ -93,9 +93,16 @@ data RuleTest = RuleTest
   { name :: String
   , input :: Object
   , output :: [Object]
+  , options :: Maybe [RuleTestOption]
   }
   deriving (Generic, FromJSON, Show)
 
+newtype RuleTestOption = TakeOne {take_one :: Bool}
```

Comment:
You should rename the constructor to `RuleTestOption` and `take_one` to `takeOne`.

Reply:
I assumed there could be more options in the future. I can rename for now, though